readthedocs/templates/projects/import_config.html

```diff
@@ -0,0 +1,55 @@
+{% extends "projects/import_base.html" %}
+{% load i18n %}
+
+{% block content %}
+<h3>{% trans "Project configuration file (<code>.readthedocs.yaml</code>)" %}</h3>
+
+<p class="info">
+    {% blocktrans trimmed %}
+    Make sure your project has a <code>.readthedocs.yaml</code> at the root of your repository. This file is required by Read the Docs to be able to build your documentation. You can <a href="https://docs.readthedocs.io/en/stable/config-file/v2.html">read more about this in our documentation</a>.
+    {% endblocktrans %}
+</p>
+
+<p class="info">
+    Here you have an example for a common Sphinx project:
+
+    <pre class="small">
+        <code>
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+    # You can also specify other tool versions:
+    # nodejs: "19"
+    # rust: "1.64"
+    # golang: "1.19"
+
+# Build documentation in the "docs/" directory with Sphinx
+sphinx:
+   configuration: docs/conf.py
+
+# Optionally build your docs in additional formats such as PDF and ePub
+# formats:
+#    - pdf
+#    - epub
+
+# Optionally, but recommended,
+# declare the Python requirements required to build your documentation
```

Comment:
For user facing parts of our product, I'd skip sembr. It looks like a typo if you aren't familiar.

```suggestion
# Optional but recommended, declare the Python requirements required to build your documentation
```

Reply:
This makes the pre block to scroll horizontally. That's why I used 2 lines. I guess that's not a problem, tho.

![Peek 2023-06-01 09-22](https://github.com/readthedocs/readthedocs.org/assets/244656/d620fcd1-d095-4e65-8449-fdb86a2af8cd)

Reply:
Ah yeah, didn't check that. Two lines is fine, split at 80 cols.

Sembr just has the habit of looking like bad beat poetry. This is totally fine for doc authoring, but doesn't look clean on user facing UI.